cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -436,6 +437,18 @@ public ClusterOperationMetadata createCluster(String name, String imageVersion,
             .build());
       }
 
+      //Set Auto Scaling Policy
+      if (conf.isAutoPolicyEnabled()) {
```

Comment:
Why we need two separate config? Just the policyURI is good enough, right? If it is not null nor empty, then we'll apply the policy

Reply:
yea we dont need it